static/app/views/preprod/buildDetails/main/insights/mainBinaryExportedSymbolsModal.tsx

```diff
@@ -0,0 +1,77 @@
+import {openInsightInfoModal} from 'sentry/actionCreators/modal';
+import {Flex} from 'sentry/components/core/layout';
+import {Text} from 'sentry/components/core/text';
+import {Heading} from 'sentry/components/core/text/heading';
+import {t, tct} from 'sentry/locale';
+import {InlineCode} from 'sentry/views/preprod/buildDetails/main/insights/insightInfoModal';
+
+function getMainBinaryExportedSymbolsContent() {
+  return (
+    <Flex direction="column" gap="2xl">
+      <Flex direction="column" gap="xl">
+        <Flex direction="column" gap="md">
+          <Heading as="h3" size="md">
+            {t('Main Binary Export Metadata')}
+          </Heading>
+          <Text>
+            {tct(
+              '[bold:What it is]: Binaries that act as entrypoints for your app, such as your main app binary or watchOS app binary, are not linked against by other binaries. This means the export trie information is unnecessary and can be removed.',
```

Comment:
I think you can just delete the "what it is" text entirely from here

the title of the modal says what it is and the text right below it implicitly is the description of it